reverse_replication/README.md

```diff
@@ -0,0 +1,60 @@
+# Reverse Replication Setup
+
+HarbourBridge currently does not support reverse replication out-of-the-box.
+The launcher.go script can be used instead to setup the resources required for a 
+reverse replication pipeline.
+
+## Resources
+The pipeline requires a few GCP resources to be setup. The launcher script creates these resources for you, skipping creation if they already exist. The resources are:
+- `Change Stream`: The target spanner database should have a changestream setup with value_capture_type = 'NEW_ROW'. This helps stream CDC events from Spanner.
+- `Ordering Dataflow Job`: This dataflow job reads from Spanner CDC, orders the data and pushes it to a PubSub topic.
+- `PubSub Topic & Subscriptions`: The topic that the ordering job pushes to needs to be created beforehand. For each shard, a subscription needs to be created, with the subscription name as the corresponding logicalShardId. These names are fetched from the source shards file mentioned later.
+- `Writer Dataflow Job`: This reads messages from the PubSub subscriptions, translates them to SQL and writes to the source shards.
+
+This launcher script will create
+
+## Arguments
+
+The script takes in multiple arguments to orchestrate the pipeline. They are:
```

Comment:
One good feature is if we can give some run mode type which can:

1. Do the full setup and launch
2. Launch only the Dataflow jobs and skip the ChangeStream and PubSub
3. Launch only the Ordering Dataflow job
4. Launch only the Writer Dataflow job

These will help in case of reruns after failure.
You could add a TODO and do this later as well.